Add unit tests for `$addLabel()` action

shay2025 opened the pull request:
This pull request implicates the following:
- add unit tests for `$addLabel()` action;
- export mocks file to an independent package and adapt the existing unit tests to that change (for now, only TestCommentOnce_XXX funcs);
- remove `$addLabel()` args len check since, in the type declaration, we already declare that this built-in must have one argument;
- add `FailOnError` function that fails and prints a message if an error occurs.

shay2025 commented:
The `FailOnError` function was removed since it was thought that it was not needed.
It was also made the reestructure of the actions builtins.

ferreiratiago commented on plugins/aladino/actions/addLabel_test.go:
```diff
@@ -0,0 +1,123 @@
+}
+
+func TestAddLabel(t *testing.T) {
+	labelA := "bug"
```
Suggestion: `label`.

shay2025 commented:
@marcelosousa @ferreiratiago the only files that introduce new code are:
- `mocks/aladino/mocks.go`: package name change and capitalize function names for export purposes.
- `plugins/aladino/actions/addLabel.go` : [Remove addLabel args len check](https://github.com/reviewpad/reviewpad/pull/32/commits/93ada2a17b8400fcf938e800c9c093c76222f4d4)
- `plugins/aladino/actions/addLabel_test.go`: unit tests for `$addLabel` action builtin - main feature
- `plugins/aladino/builtins.go`: get builtins code
- `plugins/aladino/actions/assignTeamReviewer_test.go`: added a way to go get the action builtin code by accessing the builtins structure 
https://github.com/reviewpad/reviewpad/blob/3f095bc37387f4346941a855f73dc7e8ab16430c/plugins/aladino/actions/assignTeamReviewer_test.go#L21
- `plugins/aladino/actions/commentOnce_test.go`: added a way to go get the action builtin code by accessing the builtins structure
https://github.com/reviewpad/reviewpad/blob/3f095bc37387f4346941a855f73dc7e8ab16430c/plugins/aladino/actions/commentOnce_test.go#L25

ferreiratiago reviewed (approved):
Some minor improvemts.

marcelosousa commented on plugins/aladino/actions/addLabel_test.go:
```diff
@@ -0,0 +1,123 @@
+
+	assert.Nil(t, err)
+	assert.Equal(t, wantLabels, gotLabels)
+}
```
Please add a new line here.

marcelosousa reviewed (approved):
Just a small lint issue.
